docs/source/features/sheet_management/dataframe.rst

```diff
@@ -96,6 +96,36 @@ Read a DataFrame cell:
  Edit the DataFrame
 ********************
 
+Edit Rows
+=========
+
+Edit one row.
+
+.. code:: python
+
+   df.loc[0] = {"filename": "d.jpg"}
```

Comment:
do we need to write iloc here?

Reply:
In this case, `loc` is the same as `iloc`.